source/operations/concepts/healing.rst

```diff
@@ -0,0 +1,114 @@
+.. _minio-concepts-healing:
+
+==============
+Object Healing
+==============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+What is healing?
+----------------
+
+Healing is MinIO's ability to restore an object that has been damaged, corrupted, or partially lost.
+The loss can come from multiple types of corruptions or loss, such as but not limited to:
+- drive-level errors or failure
+- OS or filesystem errors or failure
+- :ref:`bit rot <minio-ec-bitrot>`
+
+Healing and Erasure Coding
+--------------------------
+
+The ability of MinIO to restore a damaged object relates directly to the following:
+
+- total number of drives in the :ref:`erasure set <minio-erasure-coding>` where the object exists
+- number of drives available with intact parts of the object
+- :ref:`parity setting <minio-ec-parity>` for the erasure set
+
+  :term:`Parity` refers to the number of dedicated recovery shards MinIO creates when writing the object.
+  For example, an erasure set may have eight total drives and use three drives during a write for parity.
+  In this scenario, MinIO splits an object into 5 data shards and create 3 parity shards.
+  MinIO distributes these eight shards across the drives in the erasure set.
+  No one drive contains only parity shards or only data shards as a protection.
```

Comment:
```suggestion
  No one drive contains only parity shards or only data shards.
  Instead it is randomized per object to distribute reads evenly across disks.
```

Reply:
Mildly tweaked from the suggestion, but updated.